README.md

```diff
@@ -343,315 +375,340 @@ convert asff2hdf            Translate a AWS Security Finding Format JSON into a
 
 ```
-convert ckl2POAM            Translate DISA Checklist CKL file(s) to POA&M files
+convert ckl2POAM              Translate DISA Checklist CKL file(s) to POA&M files
 
-  OPTIONS
-    -O, --officeOrg=officeOrg    Default value for Office/org (prompts for each file if not set)
-    -d, --deviceName=deviceName  Name of target device (prompts for each file if not set)
-    -i, --input=input            (required) Path to the DISA Checklist File(s)
-    -o, --output=output          (required) Path to output PO&M File(s)
-    -s, --rowsToSkip=rowsToSkip  [default: 4] Rows to leave between POA&M Items for milestone
-```
+  USAGE
+    $ saf convert ckl2POAM -i <disa-checklist> -o <poam-output-folder> [-h] [-O <office/org>] [-d <device-name>] [-s <num-rows>]
 
+  FLAGS
+    -O, --officeOrg=<value>   Default value for Office/org (prompts for each file if not set)
+    -d, --deviceName=<value>  Name of target device (prompts for each file if not set)
+    -h, --help                Show CLI help.
+    -i, --input=<value>...    (required) Path to the DISA Checklist File(s)
+    -o, --output=<value>      (required) Path to output PO&M File(s)
+    -s, --rowsToSkip=<value>  [default: 4] Rows to leave between POA&M Items for milestones
 
+  ALIASES
+    $ saf convert ckl2poam
+```
 
 ##### DBProtect to HDF
-
 ```
-convert dbprotect2hdf       Translate a DBProtect report in "Check Results
-                            Details" XML format into a Heimdall Data Format JSON file
-  OPTIONS
-    -i, --input=input          'Check Results Details' XML File
-    -o, --output=output        Output HDF JSON File
+convert dbprotect2hdf         Translate a DBProtect report in "Check Results
+                              Details" XML format into a Heimdall Data Format JSON file
+  USAGE
+    $ saf convert dbprotect2hdf -i <dbprotect-xml> -o <hdf-scan-results-json> [-h]
+
+  FLAGS
+    -h, --help            Show CLI help.
+    -i, --input=<value>   (required) 'Check Results Details' XML File
+    -o, --output=<value>  (required) Output HDF JSON File
 
   EXAMPLES
-    saf convert dbprotect2hdf -i check_results_details_report.xml -o output-hdf-name.json
+    $ saf convert dbprotect2hdf -i check_results_details_report.xml -o output-hdf-name.json
 ```
 
-
 ##### Fortify to HDF
-
 ```
-convert fortify2hdf         Translate a Fortify results FVDL file into a Heimdall
-                            Data Format JSON file
-  DESCRIPTION
-    The fortify converter translates a Fortify results FVDL file (e.g., audit.fvdl)
-    into a HDF JSON. The FVDL file is an XML file that can be extracted from the
-    Fortify FPR project file using standard file compression tools.
+convert fortify2hdf           Translate a Fortify results FVDL file into a Heimdall
+                              Data Format JSON file. The FVDL file is an XML that can be
```

Comment:
use a semicolon instead of a period so that way it don't look weird that we have a period in the middle and not at the end lol